src/overlays/actors/ovl_En_Az/z_en_az.c

```diff
@@ -39,108 +80,1762 @@ const ActorInit En_Az_InitVars = {
+        { 12, { 330, 200, 200 } },
+        { 14, { 280, 150, 150 } },
+        { 16, { 330, 200, 200 } },
+    },
+    {
+        { 0, { 350, 170, 120 } },
+        { 2, { 400, 220, 170 } },
+        { 4, { 350, 170, 120 } },
+        { 6, { 400, 220, 170 } },
+        { 8, { 350, 170, 120 } },
+        { 10, { 400, 220, 170 } },
+        { 12, { 350, 170, 120 } },
+        { 14, { 400, 220, 170 } },
+        { 16, { 350, 170, 120 } },
+    },
+    {
+        { 0, { 480, 216, 156 } },
+        { 2, { 540, 216, 156 } },
+        { 4, { 450, 300, 216 } },
+        { 6, { 540, 216, 156 } },
+        { 8, { 450, 300, 216 } },
+        { 10, { 540, 216, 156 } },
+        { 12, { 450, 300, 216 } },
+        { 14, { 540, 300, 156 } },
+        { 16, { 450, 250, 216 } },
+    },
+    {
+        { 0, { 600, 375, 160 } },
+        { 2, { 640, 510, 200 } },
+        { 4, { 600, 375, 160 } },
+        { 6, { 640, 510, 200 } },
+        { 8, { 600, 375, 160 } },
+        { 10, { 640, 510, 200 } },
+        { 12, { 600, 375, 160 } },
+        { 14, { 640, 510, 200 } },
+        { 16, { 600, 375, 160 } },
+    },
+};
+static struct_80A98F94 D_80A9930C[5][4] = {
+    { { 0, 2100.0f }, { 7, 1500.0f }, { 16, 2100.0f }, { 0, 0.0f } },
+    { { 0, 1900.0f }, { 7, 2200.0f }, { 10, 1600.0f }, { 16, 1900.0f } },
+    { { 0, 1700.0f }, { 7, 2200.0f }, { 16, 1700.0f }, { 0, 0.0f } },
+    { { 0, 1900.0f }, { 4, 1600.0f }, { 10, 2200.0f }, { 16, 1900.0f } },
+    { { 0, 1900.0f }, { 7, 1400.0f }, { 16, 1900.0f }, { 0, 0.0f } },
+};
+static Vec3f D_80A993AC[3] = {
+    { 1.0f, 1.0f, 0.0f },
+    { 0.9f, 0.9f, 120.0f },
+    { 0.95f, 0.95f, 240.0f },
+};
+static Vec3f D_80A993D0[3] = {
+    { 1.5f, 1.5f, 0.0f },
+    { 1.2f, 1.2f, 120.0f },
+    { 1.35f, 1.35f, 240.0f },
+};
+static TexturePtr D_80A993F4[4] = {
+    object_az_Tex_00F918,
+    object_az_Tex_010118,
+    object_az_Tex_010918,
+    object_az_Tex_011118,
+};
+static TexturePtr D_80A99404[3] = {
+    object_az_Tex_016018,
+    object_az_Tex_016818,
+    object_az_Tex_017018,
+};
 
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/ovl_En_Az/func_80A97D5C.s")
+void EnAz_Draw(Actor* thisx, PlayState* play2) {
+    PlayState* play = play2;
+    EnAz* this = THIS;
 
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/ovl_En_Az/func_80A97E48.s")
+    OPEN_DISPS(play->state.gfxCtx);
+    POLY_OPA_DISP = Gfx_CallSetupDL(POLY_OPA_DISP, 0x19);
+    CLOSE_DISPS(play->state.gfxCtx);
 
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/ovl_En_Az/func_80A97EAC.s")
+    if (this->unk_374 & 2) {
+        SkelAnime_DrawFlexOpa(play, this->skelAnime.skeleton, this->skelAnime.jointTable, this->skelAnime.dListCount,
+                              func_80A98DA4, func_80A98E48, &this->actor);
+    } else {
+        OPEN_DISPS(play->state.gfxCtx);
+        gSPSegment(POLY_OPA_DISP++, 8, Lib_SegmentedToVirtual(D_80A993F4[this->unk_37E]));
+        gSPSegment(POLY_OPA_DISP++, 9, Lib_SegmentedToVirtual(D_80A99404[this->unk_380]));
```

Comment:
```suggestion
        gSPSegment(POLY_OPA_DISP++, 0x08, Lib_SegmentedToVirtual(D_80A993F4[this->unk_37E]));
        gSPSegment(POLY_OPA_DISP++, 0x09, Lib_SegmentedToVirtual(D_80A99404[this->unk_380]));
```
Segments are written out in full two digit hex. Could you do the other ones in this file too?